python/sglang/backend/runtime_endpoint.py

```diff
@@ -37,8 +38,7 @@ def __init__(
         self._assert_success(res)
         self.model_info = res.json()
 
-        self.chat_template = get_chat_template_by_model_path(
-            self.model_info["model_path"])
+        self.chat_template = get_chat_template_by_model_path(self.model_info["model_path"]) if "chat_template" not in kwargs else get_chat_template(kwargs["chat_template"])
```

Comment:
Do one of the following
1. Fix `get_chat_template_by_model_path`
2. Put `chat_template: Optional[str] = None`